state/state.go

```diff
@@ -67,13 +78,23 @@ func loadState(db dbm.DB, key []byte) *State {
 		}
 		// TODO: ensure that buf is completely read.
 	}
+
+	v := s.loadValidators(s.LastBlockHeight)
+	if v != nil {
+		s.lastHeightValidatorsChanged = v.LastHeightChanged
+	} else {
+		s.lastHeightValidatorsChanged = 1
```

Comment:
yep we can do that. technically there is no block 0. but just made sure we SaveValidators for height 0 when we create the state, so this can be removed